transaction-exclusion-api/app/src/integrationTest/kotlin/net/consensys/linea/transactionexclusion/TransactionExclusionAppTest.kt

```diff
@@ -197,6 +200,68 @@ class TransactionExclusionAppTest : CleanDbTestSuiteParallel() {
       )
   }
 
+  @Test
+  fun `Should save the rejected contract deployment tx from RPC`() {
+    // Save the rejected contract deployment tx from RPC (without sender address)
+    val rejectionTimeStamp = Clock.System.now()
+      .trimToMillisecondPrecision()
+      .toString()
```

Comment:
Note: it makes the test more readable if we use the real string that will be passed along e.g "2024-11-04T09:10:39:123Z"

With this `rejectionTimeStamp` I don't know exactly what is the expected String there...

Reply:
It'd not be straightforward to replace `rejectionTimeStamp` with static value here because the `transaction-exclusion-api` service running in the integration test would not retrieve rejected transaction with rejected timestamp older than 7 days